docs/src/pages/components/buttons/CustomizedButtons.js

```diff
@@ -66,6 +66,21 @@ const theme = createMuiTheme({
   palette: {
     primary: green,
   },
+  additions: {
```

Comment:
Not sure if this is the best word, but I wanted to distinguished the overrides vs "adding something new". I was thinking about variants, but we want to use this for colors too, so I don't know... 🤷‍♀️